.eslintrc.js

```diff
@@ -49,7 +49,6 @@ module.exports = {
         "quotes": ["error", "single", { "avoidEscape": true }],
```

Comment:
Maybe we need to extend from airbnb config? We will have all latest rules then.

You can apply this:
https://gist.github.com/krambertech/490f27b35d98f7d3954212d55074a547

I'll add a separate task for this